### PR TITLE
Add dimension metadata and style

### PR DESCRIPTION
**Pull request by ritvje**
- Add possibility to set dimension metadata through config
- Add possibility to set default style through config

### CODE REVIEW

File: examples/create_layers.yaml [hunk 1]
```diff
@@ -54,17 +61,29 @@ properties:
...
-  # Needs to match with properties -> indexer -> TimeAttribute
-  name: *time_attribute
-  enabled: true
-  # Images are available regularly every 5 minutes
-  presentation: DISCRETE_INTERVAL
-  # Time resolution in milliseconds.  300000 = 5 minutes
-  resolution: 300000
-  units: ISO8601
-  nearestMatchEnabled: false
+dimensions:
```
Comment (pnuu) [inline examples/create_layers.yaml:64]: If `elevation_dimension` were a main-level entry, the change would be backwards compatible with existing configs, but I see how this version is much more flexible. I think I just need to adapt my old configs to this version.

File: examples/create_layers.yaml [hunk 2]
```diff
@@ -39,6 +39,13 @@ common_items:
   # Uncomment to set cache age setting (in seconds) for HTTP header responses
   # cache_age_max: 86400
 
+default_style:
```
Comment (pnuu) [inline examples/create_layers.yaml:42]: This is a great addition! Maybe add a short comment like `# Set the default style to the layer. The style should already exist`

File: georest/__init__.py [hunk 3]
```diff
@@ -120,13 +120,22 @@ def add_layer_metadata(cat, workspace, layer_name, time_dim, meta):
...
     # Save the added metadata
     cat.save(coverage)
-    logger.info("Metadata written for layer '%s' on workspace '%s'",
-                layer_name, workspace)
+
+    if style is not None:
+        # Add default style for layer
+        layer = cat.get_layer(layer_name)
+        layer._set_default_style(cat.get_style(style["name"], workspace=style["workspace"]))
+        cat.save(layer)
```
Comment (pnuu) [inline georest/__init__.py:135]: I _think_ covering this in the tests would be as simple as setting the `default_style` in `georest.tests.test_geoserver.CREATE_LAYERS_CONFIG` dictionary and an `assert` or two to `test_create_layers()`.

Comment (pnuu) [reply georest/__init__.py:135]: Or even bette, make that a new test case based on `test_create_layers()` that only checks for the style being handled properly with an `assert` or two.

**Review by pnuu — CHANGES_REQUESTED** [review]
Very nice features! Just a small request to test that the style addition is handled correctly.

File: georest/__init__.py (hunk 3 above)
Comment (ritvje) [reply georest/__init__.py:135]: Okay, I tried to add a test case for the style (but I'm not an expert in tests so I'm not sure if this is how it should be done)

**Pull request merged**